### ISSUE TITLE
Using "rename symbol" on labelled argument breaks

### ISSUE THREAD

**Issue by JonoPrest** [rescript-lang/rescript-vscode#1004]
When renaming a labelled arg, if you include the `~` it propogates to the renamed locations. If you remove it then it does not stay a labelled argument.
https://github.com/rescript-lang/rescript-vscode/assets/65739024/6bb058e0-bcbc-4303-b6dd-c959859a13a1

**Comment by zth** [comment]
Yeah, this is known and unfortunately a little bit difficult to fix. Some WIP prior art: https://github.com/rescript-lang/rescript-vscode/pull/872

**Comment by zth** [comment]
Yeah, we definitely want it fixed, so it's great that you report it 👍
